pkg/sources/github/github.go

```diff
@@ -59,6 +58,7 @@ type Source struct {
 	repos             []string
 	orgsCache         cache.Cache
 	filteredRepoCache *filteredRepoCache
+	repoWikiCache     map[string]struct{} // repositories that have `"has_wiki": true`
```

Comment:
This is repositories that have `"has_wiki":true` _and_ the wiki was resolvable without redirecting, right? I can see myself being misled by this comment down the road - I think that just "repositories with wikis" would be sufficient, but maybe you want to be more precise.